views/building-hours/index.js

```diff
@@ -13,16 +13,29 @@ import type {BuildingInfoType} from './types'
 import hoursData from '../../data/building-hours.json'
 
 const buildingImages = {
-  pausekitchen: require('../../data/images/buildinghours/pausekitchen.small.jpeg'),
-  bookstore: require('../../data/images/buildinghours/bookstore.small.jpeg'),
-  convenience: require('../../data/images/buildinghours/convenience.small.jpeg'),
+  pause: require('../../data/images/buildinghours/pause.jpg'),
+  bookstore: require('../../data/images/buildinghours/bookstore.jpg'),
+  convenience: require('../../data/images/buildinghours/convenience.jpg'),
   postoffice: require('../../data/images/buildinghours/postoffice.small.jpeg'),
   rolvaag: require('../../data/images/buildinghours/rolvaag.small.jpeg'),
   halvorson: require('../../data/images/buildinghours/halvorson.small.jpeg'),
-  skoglund: require('../../data/images/buildinghours/skoglund.small.jpeg'),
-  cage: require('../../data/images/buildinghours/cage.small.jpeg'),
+  skoglund: require('../../data/images/buildinghours/skoglund.jpg'),
+  cage: require('../../data/images/buildinghours/cage.jpg'),
   stav: require('../../data/images/buildinghours/stav.small.jpeg'),
-  disco: require('../../data/images/buildinghours/disco.small.jpeg'),
```

Comment:
If you still have the originals, could you add them as well, please? 

ImageOptim is a nifty tool that reduces image filesize losslessly by primarily removing image metadata. It's a Mac app, but pretty handy.